src/components/ClickableStyle/ClickableStyle.module.css

```diff
@@ -478,22 +482,19 @@
 
 /**
  * Link neutral clickable style
+ * 1) Overrides the default background color on focus. Often used when the default
+ *    focus background color doesn't mesh well with the surrounding colors.
  */
 .clickable-style--link.clickable-style--neutral {
   text-decoration-color: var(--eds-theme-color-icon-neutral-strong);
 
   &:hover {
     text-decoration-color: var(--eds-theme-color-border-neutral-strong);
   }
-}
 
-/**
- * Link neutral clickable style pseudo element for the focus background state.
- * Overrides the default background color on focus. Often used when the default
- * focus background color doesn't mesh well with the surrounding colors.
- */
-.clickable-style--link.clickable-style--neutral::before {
-  background-color: var(--eds-theme-color-icon-neutral-strong);
```

Comment:
This used to be on the `::before` pseudo element all the time because that element was only visible on focus. Now that that element is gone, we apply the style to the clickable element only on focus.